lib/Runtime/HostManager/HostManager.cpp

```diff
@@ -92,6 +92,11 @@ llvm::Error HostManager::addNetwork(std::unique_ptr<Module> module,
     info.backendName = device.second->getBackendName();
     deviceInfo.push_back(info);
   }
+  // Perform a round of target-independent graph optimizations to remove unused
```

Comment:
If it's just to remove unused nodes then can't you just call DCE directly? Otherwise I would rephrase this to make it clear we're not just removing unused Nodes.

Reply:
Apparently, it is more than just DCE. DCE alone does not help that much. I'll rephrase.